### ISSUE TITLE
Can't download categories

### ISSUE THREAD

**Issue by joreilly25** [Stekeblad/Stekeblads-Video-Uploader#8]
Hey there, I was trying to change over to version 1.3 (even though I apparently hit an upload limit, so can't use the program anyway... I ended up resetting everything, and version 1.3 wasn't opening the web browser to connect to the API, so I went back to 1.2 but now it won't download categories... not sure if its on my end or not. I put in Language id = en and Country id = us.

I get "The selected language or country code is not valid or not supported by Youtube, did not receive any categories"

**Comment by Stekeblad** [comment]
Its a problem on my end. Stekeblads Video Uploader have a limit on how many operations it can perform against YouTube every day (resets at midnight pacific time). This limit has been reached several times the last week and that cause operations like retrieving categories, creating playlist and of course also uploading of videos to fail.

The good thing is that I can apply for a higher limit.

The bad thing is that the process is not that easy, I need to fill out a long form and submit several documents that someone at Google or YouTube need to review. Now that I finished the 1.3 release this is the next point I will be working on, however it will not be done in just a week or two.

Thank you for using my program, I hope you understand and will continue to use it.

**Comment by Stekeblad** [comment]
I was wrong, I have just fixed a really bad bug that broke most of the interactions against YouTube.

In the 1.3 release I tried to update the YouTube API and it did not properly update my build script. When it did not work outside my development environment I looked at my qouta limit and it was reached so I falsely assumed it was the problem.

[I have just uploaded release 1.3.1](https://github.com/Stekeblad/Stekeblads-Video-Uploader/releases/tag/1.3.1) and this time I have been more careful with my testing.

I am sorry for the problems that the broken release may have caused!
